fastapi/encoders.py

```diff
@@ -30,7 +30,7 @@ def generate_encoders_by_class_tuples(
 def jsonable_encoder(
     obj: Any,
     include: Union[SetIntStr, DictIntStrAny] = None,
-    exclude: Union[SetIntStr, DictIntStrAny] = set(),
```

Comment:
When you set "None" as default,  you will need to check condition of existence in the rest of the code.